README.md

```diff
@@ -249,17 +249,17 @@ In the above example, the ingress controller's LoadBalancer endpoint is 52.52.52
 
 #### SAS/CONNECT
 
-When running the `viya` action with `V4_CFG_CONNECT_ENABLE_LOADBALANCER=true`, a separate loadbalancer service is created to allow external SAS/CONNECT clients to connect to SAS Viya. You will need to register this LoadBalancer endpoint with your DNS provider such that the desired host name (for example, connect.example.com) points to the LoadBalancer endpoint.
+When running the `viya` action with `V4_CFG_CONNECT_ENABLE_LOADBALANCER=true`, a separate loadbalancer service is created to allow external SAS/CONNECT clients to connect to the SAS Viya platform. You will need to register this LoadBalancer endpoint with your DNS provider such that the desired host name (for example, connect.example.com) points to the LoadBalancer endpoint.
 
 
 ### Updating SAS Viya Manually
 
 Manual steps are required by the SAS software to update a SAS deployment in an existing cluster. As a result, viya4-deployment does not perform updates. The viya4-deployment tools can perform subsequent `viya,install` tasks if you are simply reapplying the same software order into the cluster.
 
-If you have an existing deployment that you performed with the viya4-deployment project, take the following steps in order to update SAS Viya:
+If you have an existing deployment that you performed with the viya4-deployment project, take the following steps in order to update the SAS Viya platform:
 
-- Follow the instructions in [Updating Software](https://go.documentation.sas.com/doc/en/sasadmincdc/default/k8sag/titlepage.htm?fromDefault=) in the SAS Viya Operations Guide.
-- You are expected to modify the steps that are described in the SAS Viya Operations Guide to accommodate the slightly different directory structure 
+- Follow the instructions in [Updating Software](https://go.documentation.sas.com/doc/en/sasadmincdc/default/k8sag/titlepage.htm?fromDefault=) in the SAS Viya platform Operations Guide.
+- You are expected to modify the steps that are described in the SAS Viya platform Operations Guide to accommodate the slightly different directory structure 
```

Comment:
SAS Viya Platform Operations Guide

Reply:
There is another one  "SAS Viya Platform Operations Guide"

Reply:
I agree, Sally. I missed these because I was in a big rush.

Reply:
Updated SAS Viya platform Operations -> SAS Viya Platform Operations